src/lexer_tests.rs

```diff
@@ -230,12 +242,39 @@ fn lex_hash() {
         unwrap_token_value(it.next()),
         TokenValue::Ident("a".to_string())
     );
-    assert_eq!(unwrap_token_value(it.next()), TokenValue::Hash);
+    let token = unwrap_token(it.next());
+    assert_eq!(token.value, TokenValue::Hash);
+    assert_eq!(
+        token.location,
+        Location {
```

Comment:
u-nit: Maybe we could have a `l` helper similar to `c` to keep things on a single line.